File: back_api/acessibilityguide_back/src/main/kotlin/com/acessibilityguide/api/model/GuideUser.kt
package com.acessibilityguide.api.model

import com.avaje.ebean.Model
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

/**
 * Created by eduardolopes on 02/06/17.
 */
@Entity
@Table(name= "guide_user")
class GuideUser: Model() {

    @Id
    var id: Long? = null

    var name: String? = null

    @Column(nullable = false, unique = true)
    var cpf: String? = null

    @Column(nullable = false, unique = true)
    var username: String? = null

    @Column(nullable = false)
    var password: String? = null

    companion object : Model.Find<Long, GuideUser>() { }

}

File: mobile/src/app/authentication/models/guideUser.ts
/**
 * Created by marcosflavio on 6/16/17.
 */
export class GuideUserModel {
    id: number;
    name: string;
    cpf: string;
    username: string;
    password: string;

    constructor(id: number, name: string, cpf: string, username: string, password: string){
        this.id = id;
        this.name = name;
        this.cpf = cpf;
        this.username = username;
        this.password = password;
    }
}